LandSandBoat/server
# Correct Spelling of Potency in comments

- No code changed at all, only comments



**_I affirm:_**

## What does this pull request do?

No code changed at all, only the spelling of Potency in the comments.

## Steps to test these changes

Run ``git grep Potentcy`` before merging this PR, see 3 results.
Run ``git grep Potentcy`` after merging this PR, see 0 results, be happy and rejoice while chanting the following:

``#NoMorePotentcyComments``
